test/JDBC/expected/BABEL-3691-vu-cleanup.out

```diff
@@ -0,0 +1,97 @@
+-- DROP TABLE
+DROP TABLE BABEL_3691_vu_prepare_t1
+GO
+
+-- Approximate numerics
+DROP TABLE BABEL_3691_vu_prepare_t2
+GO
+
+-- Date and time
+DROP TABLE BABEL_3691_vu_prepare_t3
+GO
+
+-- Character strings
+DROP TABLE BABEL_3691_vu_prepare_t4
+GO
+
+-- Unicode character strings
+DROP TABLE BABEL_3691_vu_prepare_t5
+GO
+
+-- Binary strings
+DROP TABLE BABEL_3691_vu_prepare_t6
+GO
+
+-- Return null string
+DROP TABLE BABEL_3691_vu_prepare_t7
+GO
+
+-- Rowversion and timestamp
+EXEC sp_babelfish_configure 'babelfishpg_tsql.escape_hatch_rowversion', 'ignore';
```

Comment:
Is this necessary?

Reply:
This was added with reference from other SQL test files. But yes, this is not necessary. Will remove it.